packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-out/main/aspnet/summaryofsteps.md

```diff
@@ -1,5 +1,3 @@
-## Summary of steps
-
 This use case signs a user out of the local app by revoking
 their session tokens. The sequence of steps is shown below:
```

Comment:
I think we're to use "the following" instead of "below", according to the style guide. Something like "Use the following steps..."?